hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.tsx

```diff
@@ -37,13 +38,20 @@ interface QuantitySelectorProps {
     quantity_remaining: number;
     handleChange: (e: SelectChangeEvent) => void;
     numInCart: number;
+    maxPerTeam: number | null;
 }
 
 const QuantitySelector = ({
     quantity_remaining,
     numInCart,
     handleChange,
+    maxPerTeam,
 }: QuantitySelectorProps) => {
+    const dropdownNum =
+        maxPerTeam !== null
+            ? Math.min(quantity_remaining, maxPerTeam)
+            : quantity_remaining;
+
     if (!quantity_remaining) {
```

Comment:
Could you replace quantity_remaining here with dropdownNum

Reply:
On line 55? or for all of them?

Reply:
just line 55